### DISCUSSION TITLE
How can I display tooltip on rendered link?

### DISCUSSION THREAD

**Discussion by safa-kajabi** [facebook/lexical#5017] [Q&A]
I want to display a tooltip on a rendered link in read-only mode. How can I achieve this?

![image](https://github.com/facebook/lexical/assets/127859730/6b8d1acc-996e-4171-8bf5-80415b6567af)

**Comment by SalahAdDin** [comment]
> I want to display a tooltip on a rendered link in read-only mode. How can I achieve this?
> 
> ![image](https://private-user-images.githubusercontent.com/127859730/268692766-6b8d1acc-996e-4171-8bf5-80415b6567af.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDYwNzMyMzksIm5iZiI6MTc0NjA3MjkzOSwicGF0aCI6Ii8xMjc4NTk3MzAvMjY4NjkyNzY2LTZiOGQxYWNjLTk5NmUtNDE3MS04YmY1LTgwNDE1YjY1NjdhZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNTAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDUwMVQwNDE1MzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03MjU1YTc4NzVkZjVlNzk5MGIxN2VjYmYzNDIwZDI4ZmY4MDA4MjFlMDQxMGJkYzMzOTQwNjQ2NWM4ZTFhMjRhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.87S08SNC-KUvqkow24L-xdhuR_0kvIoIE0Qa6lgen5w)

Any solution?

**Reply by moy2010** [reply]
Check the code for the playground. There's a plugin for displaying a toolbar on hover that you could tweak to fit your needs.

**Reply by SalahAdDin** [reply]
Let me check it
